src/Generator/AttributeGenerator/AttributeAssemblerBag.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Code\Generator\AttributeGenerator;
+
+final class AttributeAssemblerBag
```

Comment:
Instead of using an `AttributeAssemblerBag`, a `list<AttributeAssembler>` can be used in any downstream consumers

Reply:
no longer necessary